frontend/amundsen_application/static/js/pages/LineagePage/index.tsx

```diff
@@ -0,0 +1,247 @@
+    }
+  }
+
+  getDisplayName() {
+    const { match } = this.props;
+    const { params } = match;
+
+    return `Lineage Information | ${params.schema}.${params.table}`;
+  }
+
+  // ToDo: Should be moved to a common place
+  getTableKey() {
+    /*
+    This 'key' is the `table_uri` format described in metadataservice. Because it contains the '/' character,
+    we can't pass it as a single URL parameter without encodeURIComponent which makes ugly URLs.
+    DO NOT CHANGE
+    */
+    const { match } = this.props;
+    const { params } = match;
+
+    return `${params.database}://${params.cluster}.${params.schema}/${params.table}`;
+  }
+
+  handleClick = (e) => {
+    console.log(e);
+  };
+
+  render() {
+    // const { isLoading, statusCode, tableData } = this.props;
+    const { match, tableLineage, statusCode, isLoading } = this.props;
+    console.log('tableLineage', tableLineage);
+    console.log('statusCode', statusCode);
+
+    const { params } = match;
+    let innerContent;
+
+    // We want to avoid rendering the previous table's metadata before new data is fetched in componentDidMount
+    if (isLoading || !this.didComponentMount) {
+      innerContent = <LoadingSpinner />;
+    } else if (statusCode === SERVER_ERROR_CODE) {
+      innerContent = <ErrorMessage />;
+    } else {
+      const resourceData = {
+        database: params.database,
+        schema: params.schema,
+        name: params.table,
+        cluster: params.cluster,
+      };
+      innerContent = (
+        <div className="resource-detail-layout lineage-page">
+          <header className="resource-header">
+            <div className="header-section">
+              <span
+                className={
+                  'icon icon-header ' +
+                  getSourceIconClass(resourceData.database, ResourceType.table)
+                }
+              />
+            </div>
+            <div className="header-section header-title">
+              <h1
+                className="header-title-text truncated"
+                title={`${resourceData.schema}.${resourceData.name}`}
+              >
+                <Link to="/search" onClick={this.handleClick}>
+                  {resourceData.schema}
+                </Link>
+                .{resourceData.name}
+                <span className="text-secondary lineage-graph-label">
+                  Lineage Graph
+                </span>
+              </h1>
+              <div className="body-2">
+                <Link
+                  className="resource-list-item table-list-item"
+                  to={getLink(resourceData, 'table-lineage-page')}
+                >
+                  Back to table details
+                </Link>
+              </div>
+            </div>
+            <div className="header-section header-links">
+              <button
+                type="button"
+                className="btn btn-close clear-button icon-header"
+              />
```

Comment:
Buttons need to have some text, if it shouldn't be visible, then add the `sr-only` class so it is still accessible.